R/calculate_annual_pd_changes.R

```diff
@@ -59,6 +59,8 @@ calculate_pd_change_annual <- function(data,
     dplyr::group_by(
       .data$scenario_name, .data$scenario_geography, .data$investor_name,
       .data$portfolio_name, .data$id, .data$company_name, .data$ald_sector,
+      # TODO: clarify the appropriate summation for annual PD calculations
```

Comment:
still relevant?

Reply:
not for this PR, but still relevant to clarify in general